tracer/src/Datadog.Trace/LibDatadog/TraceExporterConfiguration.cs

```diff
@@ -105,6 +105,15 @@ public string? Service
         }
     }
 
+    public TelemetryClientConfiguration TelemetryClientConfiguration
+    {
+        init
+        {
+            using var error = NativeInterop.Config.EnableTelemetry(this, value);
```

Comment:
Is there any reason that this wouldn't be on by default?

Reply:
It is enabled with the creation of the trace exporter.

Could you expand what do you mean by default here?

Reply:
> Could you expand what do you mean by default here?

I mean, do we need this call at all? Shouldn't it be on by default?

Reply:
I guess the answer is "yes, theoretically, but there are required settings for telemetry, so in practice, no". So ignore me 😄

Reply:
it is just that telemetry part of exporter config, always setting it whether enabled or not. 

But yes, by default it is enabled in the native side.